.github/workflows/python-code-cov.yml

```
@@ -21,7 +24,7 @@ jobs:
     - name: Set up Python 3.10
       uses: actions/setup-python@v3
```

actionlint (1.7.9)

the runner of "actions/setup-python@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)